spec/models/proposal_spec.rb

```diff
@@ -332,4 +332,13 @@
       expect(observation_creator_double).to have_received(:run)
     end
   end
+
+  describe "#tags" do
+    it "can add case-insensitive tags" do
```

Comment:
I think it's also possible to use space separators rather than comma with this lib -- maybe while we're writing this test we can also add that so the description is "adds case-insensitive tags via comma separated strings" ?